### DISCUSSION TITLE
Como funciona o buscador de videos do Discord (BOT)

### DISCUSSION THREAD

**Discussion by joaogm1** [frontendbr/forum#1850] [Perguntas e Respostas]
Gente, estou com uma dúvida, sou um novato na área da programação e e estou tentando treinar criando projetos pessoais. Esses dias estava no discrod e percebi a maneira como o robo busca as músicas do youtube automaticamente, alguém sabe me dizer como isso é feito? e como replicar em código?

obs: No caso quero saber como o bot busca e adiciona para a playlist

**Comment by ninetails** [comment]
Utilize a busca do próprio GitHub:

> https://github.com/search?q=bot+discord

E aí vai filtrando pelo que vc precisa, ex: https://github.com/search?q=bot+discord+youtube e aí filtrando por JavaScript, etc
